server/serverbase.js

```diff
@@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
       }
     }
   }
+  function loadThumb() {
+    return function(req, res, next) {
+      req.url = path.dirname(req.url) + '/.thumbs/' +
```

Comment:
You used a THUMB_DIR constant below - maybe you should pull it global and use it everywhere.